_services/bsp-builds-support.md

```diff
@@ -1,47 +1,51 @@
 ---
-title: AOSP and Linux BSPs, Builds and Support
-jumbotron:
-    title: AOSP and Linux BSPs, Builds and Support
-    title-class: my-3
-    class: dev_services text-center
-    image: /assets/images/content/IoT_Planet_UNDER_2MB.jpg
-    buttons:
-      - title: Contact Us
-        url: "#contact_form"
-        style: btn btn-primary btn-lg my-md-3 d-none d-md-inline-block text-uppercase ds_contact_form_btn
-      - title: Contact Us
-        url: "#contact_form"
-        style: btn btn-primary btn-sm my-2 d-inline-block d-md-none text-uppercase ds_contact_form_btn
+title: Linux  & Android Boards Support Package. Builds and Support.
```

Comment:
The change to the title will result in the services landing page not showing the related icons/services as these are referenced by their title. I will need to push an update to stop this happening.